spec/features/admin/shipping_methods_spec.rb

```diff
@@ -88,6 +89,7 @@
       within(".side_menu") do
         click_link "Shipping Methods"
       end
+      DefaultShippingCategory.find_or_create
```

Comment:
Is this line needed?

Reply:
Hi @Matt-Yorkley thanks for taking a look. This line was a hacky way to add a second default category to the one already provided by the Spree factory. I've added a new commit that overrides the Spree factory and cleans up the spec.